source/module_io/json_output/abacusjson.h

```diff
@@ -76,6 +66,41 @@ class AbacusJson
 
 
 
+    /**
+     *  @brief: The template specialization method adds value to the doc tree
+     *
+     *  @param: 'value' is the value that needs to be added to the json tree, which type can be 
+     *          Json::jsonValue or other common value type(such as int, double ,bool ,std::string).
+     *
+     *          'IsArray' is a bool value, means the whether the root node to which 'value' is added is an array.
+     *
+
+                'args' is a parameter list, representing the path to add json.
+     *  @usage: 1. Add/Modify a double val to object json node (key2 is a object node): 
+     *                Json::AbacusJson::add_json(3.1415,false,"key1","key2");
+     *
+     *          2. Pushback a double val to array json node (key2 is a array node):
+     *                Json::AbacusJson::add_json(3.1415,true, "key1","key2");
+     *
+     *          3. Modify a doble val to array json node (key2 is a array node), when use this method, 
+     *            The index number of the array starts at 1, and if the index is 0, it means that the last element 
+     *            of the array is modified:
```

Comment:
The index is started from 0, and -1 means the last element?

Reply:
Yes, I forgot to change the doc...